lib/src/main/scala/spinal/lib/bus/amba3/ahblite/AhbLite3SlaveFactory.scala

```diff
@@ -42,6 +42,13 @@ class AhbLite3SlaveFactory(bus: AhbLite3, incAddress: Int = 0) extends BusSlaveF
   override def readAddress()  = bus.HADDR
   override def writeAddress() = bus.HADDR
 
+  // TODO: Implement
+  val readErrorFlag = False
+  val writeErrorFlag = False
+
+  override def readError(): Unit = readErrorFlag := True
+  override def writeError(): Unit = writeErrorFlag := True
```

Comment:
Side note: I love this way to do!

When I see this kind of code, I want to add an "idioms" (as in "copy-and-swap" C++ idiom) part in RTD, full of stuff about building useful semantics.

This would be named like "muxing function idiom" or "signal-function idiom".